README.md

```diff
@@ -3,37 +3,33 @@
 [![Test coverage](https://codecov.io/gh/ComPWA/expertsystem/branch/master/graph/badge.svg)](https://codecov.io/gh/ComPWA/expertsystem)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/db355758fb0e4654818b85997f03e3b8)](https://www.codacy.com/gh/ComPWA/expertsystem)
 [![Documentation build status](https://readthedocs.org/projects/expertsystem/badge/?version=latest)](https://pwa.readthedocs.io/projects/expertsystem/)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen)](https://github.com/pre-commit/pre-commit)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
 # PWA Expert System
 
-The goal is to build state transition graphs, going from an initial state to a
```

Comment:
I would add a small usage section and refer to the jupyter notebook.

Reply:
I would only add a reference to the notebooks under the workflow section

Reply:
exactly

Reply:
So I'm not entirely sure yet how useful such a note would be, as the Sphinx docs (will) speak for themselves. Most I can think of is

> The [usage page](https://pwa.readthedocs.io/projects/expertsystem/en/latest/usage.html) is a helpful start in particular.

But the wording here is incorrect (the `index.rst` describes intention already sufficiently).

Reply:
So up to you @spflueger, can append it to your upcoming commits

Reply:
Fair point, ok then lets leave it like this.